From c721f190587402c51fcc36197e7ebea0bab7ffb0 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Wed, 20 Apr 2016 00:06:01 +0800
Subject: [PATCH 1152/1240] a3700: clean defconfig of non dram init option for
 SPL

In the future, SPL would only be used for DRAM init debug,
and would not be build into u-boot image in normal case.
This patch disables all the unrelated option for SPL.

Change-Id: I198a722fd9e16f2bced58fcb15316c0410fe0717
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/29181
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
---
 configs/mvebu_armada3700_defconfig | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/configs/mvebu_armada3700_defconfig b/configs/mvebu_armada3700_defconfig
index abd769b..306c3e5 100644
--- a/configs/mvebu_armada3700_defconfig
+++ b/configs/mvebu_armada3700_defconfig
@@ -3,12 +3,11 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_3700=y
 CONFIG_MVEBU_A3700_IO_ADDR_DEC=y
-+S:CONFIG_MVEBU_A3700_MISC_INIT=y
-+S:CONFIG_MVEBU_A3700_CLOCK=y
+CONFIG_MVEBU_A3700_CLOCK=y
 CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_A3700_UART=y
 CONFIG_XENON_MMC=y
-+S:CONFIG_DEVEL_BOARD=y
+CONFIG_DEVEL_BOARD=y
 CONFIG_CMD_BDI=y
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
@@ -32,13 +31,12 @@ CONFIG_DM=y
 CONFIG_MVEBU_ADVK_PCIE=y
 CONFIG_MV_INCLUDE_SATA=y
 CONFIG_SCSI_AHCI_PLAT=y
-+S:CONFIG_MVEBU_MBUS=y
-+S:CONFIG_MVEBU_MBUS_SPL_ONLY=y
+CONFIG_MVEBU_MBUS=y
 CONFIG_MVNETA=y
 CONFIG_MVEBU_NETA_BYPASS_DEC_WIN=y
 CONFIG_PHY_MARVELL=y
-+S:CONFIG_I2C_MV=y
-+S:# CONFIG_I2C_MV_PAD_REG is not set
+CONFIG_I2C_MV=y
+# CONFIG_I2C_MV_PAD_REG is not set
 CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_PINCTL=y
 CONFIG_MVEBU_GPIO=y
-- 
1.9.1

